src/stirling/source_connectors/socket_tracer/uprobe_manager.cc

```diff
@@ -551,7 +566,7 @@ namespace {
 
 // Convert PID list from list of UPIDs to a map with key=binary name, value=PIDs
 std::map<std::string, std::vector<int32_t>> ConvertPIDsListToMap(
-    const absl::flat_hash_set<md::UPID>& upids) {
+    const absl::flat_hash_set<md::UPID>& upids, const std::vector<std::string>& binary_filter) {
```

Comment:
nit/naming: `binary_filter` -> `opt_out_filter` for clarity.

Reply:
Renamed in [1207906d583090c861d06971e20a6da9c38d69fd](https://github.com/pixie-io/pixie/pull/1971/commits/1207906d583090c861d06971e20a6da9c38d69fd).